dts/bindings/phy/can-transceiver.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2022 Vestas Wind Systems A/S
+# SPDX-License-Identifier: Apache-2.0
+
+# Common fields for CAN transceivers
+
+include: phy-controller.yaml
```

Comment:
This is not existing, as far as I can see.

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/phy/phy-controller.yaml ?

Reply:
Sorry, never mind. For some reason didn't find it during my search through the Zephyr tree.